snippets/worker-latency-profile-intro.adoc

```diff
@@ -8,16 +8,20 @@
 
 :_content-type: SNIPPET
 
-All nodes send heartbeats to the Kubernetes Controller Manager Operator (kube controller) in the {product-title} cluster every 10 seconds, by default. If the cluster does not receive heartbeats from a node, {product-title} responds using several default mechanisms.
+All nodes send heartbeats to the Kubernetes Controller Manager (kube controller) in the {product-title} cluster every 10 seconds, by default. If the cluster does not receive heartbeats from a node, {product-title} responds using several default mechanisms.
```

Comment:
Kubelet doesn't send heartbeats to KCM.

Reply:
@harche Thank you, I agree!. The kublet sets a status which KCM reads every 10 seconds. Do you agree that the word "heartbeat" is not correct, and that the word "status" IS correct? If so would you agree to the following change:

Reply:
You can see the new preview at: https://60109--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/scaling-worker-latency-profiles.html